requests/request.go

```diff
@@ -50,12 +50,16 @@ func NewPostRequest(apiURI, path string, requestBody interface{}) (*http.Request
 
 // NewSignedRequest signs payload and generates http request
 func NewSignedRequest(httpMethod, apiURI, path string, requestBody interface{}, signer identity.Signer) (*http.Request, error) {
-	bodyBytes, err := encodeToJSON(requestBody)
-	if err != nil {
-		return nil, err
+	var bodyBytes []byte = nil
```

Comment:
I'd name it just `body`, we don't need to include type into the variable name.

Reply:
Renamed to `encodedBody`, because we have `requestBody` in the same scope, so having `requestBody` and `body` in the same scope seemed confusing.